src/components/Contacts/ContactDetails/ContactReferralTab/ContactReferralTab.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  Box,
+  Paper,
+  styled,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Contact } from '../../../../../graphql/types.generated';
+import { useContactReferralTabQuery } from './ContactReferralTab.generated';
+
+const ContactReferralContainer = styled(Box)(({ theme }) => ({
+  margin: theme.spacing(1),
+}));
+
+const ContactReferralLoadingPlaceHolder = styled(Skeleton)(({ theme }) => ({
+  width: '100%',
+  height: '24px',
+  margin: theme.spacing(2, 0),
+}));
+
+interface ContactReferralTabProps {
+  accountListId: string;
+  contactId: string;
+}
+
+export const ContactReferralTab: React.FC<ContactReferralTabProps> = ({
+  accountListId,
+  contactId,
+}) => {
+  const { data, loading } = useContactReferralTabQuery({
```

Comment:
Should we have something for errors?

Reply:
I'll make a Jira ticket. I think we can handle it globally.